tests/test_bench.py

```diff
@@ -326,7 +331,8 @@ def test_unidirectional_coupling_map() -> None:
         device_name="oqc_lucy",
     )
     # check that all gates in the circuit are in the coupling map
-    assert qc.valid_connectivity(arch=Architecture(utils.get_cmap_oqc_lucy()), directed=True)
+    cmap_converted = [(i, j) for (j, i) in utils.get_cmap_oqc_lucy()]
+    assert qc.valid_connectivity(arch=Architecture(cmap_converted), directed=True)
```

Comment:
Same question as before: Why the change here?

Reply:
same as above